src/substrait/textplan/SymbolTablePrinter.cpp

```diff
@@ -432,6 +435,199 @@ std::string SymbolTablePrinter::outputToText(const SymbolTable& symbolTable) {
   return text.str();
 }
 
+void SymbolTablePrinter::addInputsToRelation(
+    const SymbolInfo& symbolInfo,
+    ::substrait::proto::Rel* relation) {
+  auto relationData = ANY_CAST(std::shared_ptr<RelationData>, symbolInfo.blob);
+
+  // Connect up the incoming inputs in the relation data to the appropriate
+  // input/left/right/inputs of this relation (which recursively also needs to
+  // have its inputs added).
+  if (symbolInfo.subtype.type() != typeid(RelationType)) {
+    // The type isn't one we expected bail.
+    return;
```

Comment:
We might consider some kind of `UNREACHABLE` macro which does nothing in release mode but throws an exception (or asserts with the line number) in debug mode.